lib/Yam/agama/agama_base.pm

```diff
@@ -36,7 +36,7 @@ sub test_flags {
 }
 
 sub upload_traces {
```

Comment:
this class still contains the change of password which doesn't make sense anymore, you could clean some code here.
Additionally we could remove this needle not needed here: https://openqa.opensuse.org/tests/4441057#step/patch_agama_tests/1 (wherever it is).

Reply:
Solved aca47525aabf6b10f5285e7a04d1d88eee7da4d4